features/Steps/LabelSteps.php

```diff
@@ -34,6 +34,58 @@ public function iCreateAnInvisibleLabelWithARandomNameOfCharacters(int $nrOfChar
         );
     }
 
+    /**
+     * @Given labels test data is available
+     */
+    public function labelsTestDataIsAvailable(): void
+    {
+        // Create "public-visible" if it doesn't exist yet and (re)set the right privacy and visibility in case its needed
+        $this->createLabel('public-visible', true, true);
```

Comment:
I would first check if the label exists and only create it when it does not exist. When it already exists there is no need to create it again, only making sure to set the correct privacy and visibility.